bambi/tests/test_built_models.py

```diff
@@ -368,47 +275,45 @@ def test_cell_means_with_random_intercepts(crossed_data):
 
     # check that fixed effect design matries are the same,
     # even if term names / level names / order of columns is different
-    X0 = set([tuple(t.data[:,lev]) for t in model0.fixed_terms.values() for lev in range(len(t.levels))])
-    X1 = set([tuple(t.data[:,lev]) for t in model1.fixed_terms.values() for lev in range(len(t.levels))])
+    X0 = set([tuple(t.data[:, lev]) for t in model0.fixed_terms.values()
+              for lev in range(len(t.levels))])
+    X1 = set([tuple(t.data[:, lev]) for t in model1.fixed_terms.values()
+              for lev in range(len(t.levels))])
     assert X0 == X1
 
-    # check that add_formula and add_term models have same priors for fixed effects
-    priors0 = {x.name:x.prior.args for x in model0.terms.values() if not x.random}
-    priors1 = {x.name:x.prior.args for x in model1.terms.values() if not x.random}
+    # check that add_formula and add_term models have same priors for fixed
+    # effects
+    priors0 = {
+        x.name: x.prior.args for x in model0.terms.values() if not x.random}
+    priors1 = {
+        x.name: x.prior.args for x in model1.terms.values() if not x.random}
     assert set(priors0) == set(priors1)
 
-    # check that add_formula and add_term models have same priors for random effects
-    priors0 = {x.name:x.prior.args['sd'].args for x in model0.terms.values() if x.random}
-    priors1 = {x.name:x.prior.args['sd'].args for x in model1.terms.values() if x.random}
+    # check that add_formula and add_term models have same priors for random
+    # effects
+    priors0 = {x.name: x.prior.args[
+        'sd'].args for x in model0.terms.values() if x.random}
+    priors1 = {x.name: x.prior.args[
+        'sd'].args for x in model1.terms.values() if x.random}
     assert set(priors0) == set(priors1)
 
     # test summary
-    # it looks like some versions of pymc3 add a trailing '_' to transformed vars and
-    # some dont. so here for consistency we strip out any trailing '_' that we find
-    full = fitted.summary(exclude_ranefs=False, hide_transformed=False).index
-    full = set([re.sub(r'_$', r'', x) for x in full])
-    test_set = fitted.summary(exclude_ranefs=False).index
-    test_set = set([re.sub(r'_$', r'', x) for x in test_set])
-    assert test_set == full.difference(set(['Y_sd_interval','u_subj_sd_log']))
-    test_set = fitted.summary(hide_transformed=False).index
-    test_set = set([re.sub(r'_$', r'', x) for x in test_set])
-    assert test_set == full.difference(set(['subj[{}]'.format(i) for i in range(10)]))
-
-    # test get_trace
-    test_set = fitted.get_trace().columns
-    test_set = set([re.sub(r'_$', r'', x) for x in test_set])
-    assert test_set == full.difference(set(['Y_sd_interval','u_subj_sd_log']))\
-        .difference(set(['subj[{}]'.format(i) for i in range(10)]))
+    fitted.summary()
```

Comment:
You don't like the tests that `exclude_ranefs` and `hide_transformed` remove all/only the variables they're supposed to remove?